deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -0,0 +1,1135 @@
+from collections import OrderedDict
+
+from typing import Callable, Optional, List, Dict
+from mmdet.apis.train import *
+from mmcv.utils import build_from_cfg
+from mmdet.datasets.builder import PIPELINES
+from mmdet.datasets.pipelines import Compose
+from mmcv.parallel import collate
+from functools import partial
+from deeplake.integrations.pytorch.dataset import TorchDataset
+from deeplake.client.client import DeepLakeBackendClient
+from mmdet.core import BitmapMasks
+import deeplake as dp
+from deeplake.util.warnings import always_warn
+import os.path as osp
+import warnings
+from collections import OrderedDict
+import mmcv
+from mmcv.runner import init_dist
+import torch
+import numpy as np
+from mmcv.utils import print_log
+from terminaltables import AsciiTable
+from mmdet.core import eval_map, eval_recalls
+from mmdet.datasets.pipelines import Compose
+from mmdet.utils.util_distribution import *
+import tempfile
+from deeplake.integrations.mmdet import mmdet_utils
+from deeplake.enterprise.dataloader import indra_available, dataloader
+from PIL import Image, ImageDraw
+import os
+
+
+class Dummy:
+    pass
+
+
+def force_cudnn_initialization(device_id):
```

Comment:
ton of utility functions on top. Can we move them to some util files? Preferably not the only existing one, some util files for pascal format, etc